ios/SalesforceReactTests/package.json

```diff
@@ -9,22 +9,23 @@
         "SalesforceMobileSDK-iOS": "https://github.com/forcedotcom/SalesforceMobileSDK-iOS.git#dev"
     },
     "dependencies": {
-        "@react-native-community/cli": "^4.5.1",
-        "acorn": "^6.4.1",
-        "chai": "4.1.2",
-        "react-native-force": "git+https://github.com/forcedotcom/SalesforceMobileSDK-ReactNative.git#dev",
-        "eslint": "^5.1.0",
-        "react": "16.11.0",
-        "react-native": "0.62.2"
+        "react": "16.13.1",
+        "react-native": "0.63.2",
+        "react-native-force": "git+https://github.com/wmathurin/SalesforceMobileSDK-ReactNative.git#rn63",
```

Comment:
Will switch to git+https://github.com/forcedotcom/SalesforceMobileSDK-ReactNative.git#dev before merging